src/cmd/INIT/ratz.c

```diff
@@ -93,7 +93,7 @@ static const char usage[] =
 
 #else
 
-#if !defined(_WINIX) && (_UWIN || __CYGWIN__ || __EMX__)
```

Comment:
FYI, This is part of the legacy build system that we no longer support. It simply hasn't been removed as switching from Nmake to Meson is a work in progress.